<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSpatialQueryDialogBase</class>
 <widget class="QDialog" name="QgsSpatialQueryDialogBase">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Spatial Query</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="lytOperatorResultInvalid">
     <item>
      <layout class="QVBoxLayout" name="lytOperator">
       <item>
        <widget class="QGroupBox" name="grpTargetGroupBox">
         <property name="toolTip">
          <string>Layer on which the topological operation will select geometries</string>
         </property>
         <property name="title">
          <string>Target layer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QComboBox" name="targetLayerComboBox">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Select the target layer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="usingSelectedTargetCheckBox">
            <property name="toolTip">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;When checked the operation will only consider selected geometries of the target layer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Selected feature(s) only</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>usingSelectedTargetCheckBox</zorder>
         <zorder>targetLayerComboBox</zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpReferenceGroupBox">
         <property name="toolTip">
          <string>Layer whose geometries will be used as reference by the topological operation</string>
         </property>
         <property name="title">
          <string>Reference layer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QComboBox" name="referenceLayerComboBox">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Select the reference layer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="usingSelectedReferenceCheckBox">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>18</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;When checked the operation will be only consider selected geometries of the reference layer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Selected feature(s) only</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpOperationGroupBox">
         <property name="toolTip">
          <string>Topological operations between layers of target and reference</string>
         </property>
         <property name="title">
          <string>Topological operation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QComboBox" name="operantionComboBox">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Select the topological operation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lytResultInvalidMessage">
       <item>
        <layout class="QHBoxLayout" name="lytResultInvalid">
         <item>
          <widget class="QGroupBox" name="grpResult">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>Results</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="lytTargetResult">
              <item>
               <widget class="QPushButton" name="pushButtonSelectResultTarget">
                <property name="toolTip">
                 <string>Select all features of result</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="qgsspatialquerydialogbase.qrc">
                  <normaloff>:/icons/selectall.png</normaloff>:/icons/selectall.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="resultTargetLabel">
                <property name="minimumSize">
                 <size>
                  <width>18</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Total of features from query</string>
                </property>
                <property name="text">
                 <string>Target layer</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="resultFeatureTargetListWidget">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Select item to identify geometry of feature</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpInvalid">
           <property name="title">
            <string>Invalid features</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="lytTargetInvalid">
                <item>
                 <widget class="QPushButton" name="pushButtonSelectInvalidTarget">
                  <property name="toolTip">
                   <string>select all features invalid target</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="qgsspatialquerydialogbase.qrc">
                    <normaloff>:/icons/selectall.png</normaloff>:/icons/selectall.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="invalidTargetLabel">
                  <property name="text">
                   <string>Target layer</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="invalidFeatureTargetListWidget">
                <property name="toolTip">
                 <string>Select item to identify geometry of feature</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="lytReferenceInvalid">
                <item>
                 <widget class="QPushButton" name="pushButtonSelectInvalidReference">
                  <property name="toolTip">
                   <string>select all features invalid reference</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="qgsspatialquerydialogbase.qrc">
                    <normaloff>:/icons/selectall.png</normaloff>:/icons/selectall.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="invalidReferenceLabel">
                  <property name="text">
                   <string>Reference layer</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="invalidFeatureReferenceListWidget">
                <property name="toolTip">
                 <string>Select item to identify geometry of feature</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lytShowInfo">
         <item>
          <widget class="QCheckBox" name="ckboxLogProcessing">
           <property name="toolTip">
            <string>Check to show log processing of query</string>
           </property>
           <property name="text">
            <string>Show log messages</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spaceShowInfo">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelInfo">
           <property name="text">
            <string>* Select item to highlight on map</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditStatus">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="lytProgressBarButtonsMain">
     <item>
      <widget class="QProgressBar" name="progressBarStatus">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBoxMain">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Run query or close the window</string>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>grpTargetGroupBox</zorder>
  <zorder>grpReferenceGroupBox</zorder>
  <zorder>grpOperationGroupBox</zorder>
  <zorder>progressBarStatus</zorder>
  <zorder>buttonBoxMain</zorder>
  <zorder>grpResult</zorder>
  <zorder>grpInvalid</zorder>
  <zorder>grpInvalid</zorder>
  <zorder>textEditStatus</zorder>
  <zorder>ckboxLogProcessing</zorder>
  <zorder>labelInfo</zorder>
 </widget>
 <tabstops>
  <tabstop>targetLayerComboBox</tabstop>
  <tabstop>usingSelectedTargetCheckBox</tabstop>
  <tabstop>referenceLayerComboBox</tabstop>
  <tabstop>usingSelectedReferenceCheckBox</tabstop>
  <tabstop>operantionComboBox</tabstop>
  <tabstop>resultFeatureTargetListWidget</tabstop>
  <tabstop>invalidFeatureTargetListWidget</tabstop>
  <tabstop>invalidFeatureReferenceListWidget</tabstop>
  <tabstop>buttonBoxMain</tabstop>
 </tabstops>
 <resources>
  <include location="qgsspatialquerydialogbase.qrc"/>
 </resources>
 <connections/>
</ui>
